purchase_request_type/views/purchase_request_type_view.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_purchase_request_type_tree" model="ir.ui.view">
+        <field name="model">purchase.request.type</field>
+        <field name="arch" type="xml">
+            <tree string="Purchase Request Types" decoration-muted="active == False">
+                <field name="sequence" widget="handle" />
+                <field name="active" invisible="True" />
+                <field name="name" />
+                <field name="sequence_id" />
+                <field name="picking_type_id" />
+                <field name="description" />
+                <field name="company_id" groups="base.group_multi_company" />
+            </tree>
+        </field>
+    </record>
+    <record id="view_purchase_request_type_form" model="ir.ui.view">
+        <field name="model">purchase.request.type</field>
+        <field name="arch" type="xml">
+            <form string="Type">
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            name="toggle_active"
```

Comment:
toggle_active no need in v.14, archived button is in under action button
![Selection_291](https://user-images.githubusercontent.com/24691983/112118119-a1ebe200-8bee-11eb-89d6-ef6c7abb92a9.png)

**example fix**
https://github.com/OCA/purchase-workflow/pull/1147/files#diff-6155a254e04cee4f8c777946add4303b338735a57f7bd4ee0cedd1d1bfab9ae1L23